crates/compiler/parse/src/expr.rs

```diff
@@ -1586,12 +1127,13 @@ fn finish_parsing_alias_or_opaque<'a>(
             }
         }
 
-        match kind {
+        match kind.value {
             AliasOrOpaque::Alias => {
-                let (_, signature, state) =
-                    alias_signature_with_space_before().parse(arena, state, min_indent)?;
+                let (_, signature, state) = alias_signature().parse(arena, state, min_indent)?;
 
-                let def_region = Region::span_across(&expr.region, &signature.region);
+                // TODO: this code used to be broken and it dropped the spaces after the operator.
+                // The formatter is not expecting this, so let's keep it as is for now.
+                // let signature = signature.map(|v| v.maybe_before(arena, spaces_after_operator));
```

Comment:
So the plan is to make this change in a later PR? I guess that means all the changes in formatting are from just this change then.

Reply:
Hmm; the intent of having this commented out was to _not_ cause formatting changes. It's possible that didn't quite work out, or is also possible that the formatting changes you're seeing are unrelated.